packages/ui/src/components/accordion/accordian.native.tsx

```diff
@@ -0,0 +1,28 @@
+import { ReactNode, useState } from 'react';
+
+import { useTheme } from '@shopify/restyle';
+
+import { ChevronDownIcon, ChevronUpIcon, Text, Theme, TouchableOpacity } from '../../../native';
+
+// TODO: This should use the Cell component?
+export function Accordian({ content, label }: { content: ReactNode; label: string }) {
```

Comment:
_nit_ here but the folder is spelt `accordion` and the file + component is `Accordian` so it should be consistent and use `Accordion`